linker/Linker/AssemblyResolver.cs

```diff
@@ -55,12 +58,32 @@ public AssemblyResolver (Dictionary<string, AssemblyDefinition> assembly_cache)
 			_assemblies = assembly_cache;
 		}
 
+		public bool IgnoreUnresolved {
+			get { return _ignoreUnresolved; }
+			set { _ignoreUnresolved = value; }
+		}
+
+		public LinkContext Context {
+			get { return _context; }
+			set { _context = value; }
+		}
+
 		public override AssemblyDefinition Resolve (AssemblyNameReference name, ReaderParameters parameters)
```

Comment:
I think it'd be better to pass LinkContext Context as the parameter

Reply:
The problem is that the Resolve method signature is part of IAssemblyResolver, defined in cecil. I want warnings to be logged when an assembly can't be resolved while trying to resolve TypeReferences to TypeDefinitions using cecil, for example. Do you have any suggestions?

Reply:
hmm, I see